Units/parser-javascript.r/js-many-functions.d/expected.tags

```diff
@@ -0,0 +1,16 @@
+AnonymousFunction8797a7c50100	input.js	/^  return function(x) {$/;"	f	function:f1
+C1	input.js	/^var C1 = {$/;"	c
+K1	input.js	/^class K1 {$/;"	c
+a	input.js	/^function a() {$/;"	f
+b	input.js	/^  function b() {$/;"	f	function:a
+c	input.js	/^    function c() {$/;"	f	function:a.b
```

Comment:
Following question is to myself but I would like to hear your comment.

For c, a.b is set as scope. None defines what is "scope". However, just one upper level of c is b.
So I think just printing b is enough.

I know there many inconsistency in ctags about this topic. How do you think?